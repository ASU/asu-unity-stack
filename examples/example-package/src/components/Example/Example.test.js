import React from 'react'
import { render} from '@testing-library/react'
import Example from './Example'



describe('The Example', () => {
    it('renders the component', () => {

      const container = render(<Example  {...{items: [{title: "test"}]}} />);
            expect(container.firstChild).toMatchSnapshot()
    })
});
