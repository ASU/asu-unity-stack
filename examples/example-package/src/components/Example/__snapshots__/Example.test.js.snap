// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Example renders the component 1`] = `undefined`;
